### DISCUSSION TITLE
Low ISO Priority Script

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#1205] [General]
For action sports and general stabilization purposes the camera exposure rules are generally good.  But if you are using a gimbal, firmly attached to a heavy vehicle, you want the ISO to low as possible, without having the manual set ISO Max to 100. Setting ISO Max of 100 works fine, until you walk the camera from daylight outside into a dark interior. This script keeps the camera exposure fully automatic, but biases toward a lower ISO.

Very experimental.  Tested on HERO11, 12 and 13 only.
![Low ISO Priority-12_17_2024, 18_15_51](https://github.com/user-attachments/assets/d8317907-028f-467d-9f6b-4a070655c895)

This is code in the Blockly script builder
![image](https://github.com/user-attachments/assets/8d704148-6db0-4cce-9718-fe66f14db230)

**Comment by markusfinholt** [comment]
Does this work together with the EXPX setting?
I've had great results having the max shutter to 1/180 for stabilization when mountain biking.

**Reply by dnewman-gpsw** [reply]
No, I don't think it would work together. If you set the above script to use =M180 it would set 1/180 as the maximum exposure length.  The above is more for non-action shooting when you want a ISO bias, rather than shutter bias that modern GoPro typically default to.

**Comment by PaulTheShort** [comment]
Thanks David! That's extremely helpful. I'll test it some more today.
